README.md

```diff
@@ -99,10 +99,10 @@ Curious to learn more about GPT-4 Vision? [Check out our GPT-4V experiments 🧪
-| [YOLOX Object Detection](https://github.com/roboflow-ai/notebooks/blob/main/notebooks/train-yolox-object-detection-on-custom-data.ipynb) | [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/roboflow-ai/notebooks/blob/main/notebooks/train-yolox-object-detection-on-custom-data.ipynb) [![Kaggle](https://kaggle.com/static/images/open-in-kaggle.svg)](https://kaggle.com/kernels/welcome?src=https://github.com/roboflow-ai/notebooks/blob/main/notebooks/train-yolox-object-detection-on-custom-data.ipynb)  | [![Roboflow](https://raw.githubusercontent.com/roboflow-ai/notebooks/main/assets/badges/roboflow-blogpost.svg)](https://blog.roboflow.com/how-to-train-yolox-on-a-custom-dataset) [![YouTube](https://badges.aleen42.com/src/youtube.svg)](https://www.youtube.com/watch?v=q3RbFbaQQGw) | [![GitHub](https://badges.aleen42.com/src/github.svg)](https://github.com/Megvii-BaseDetection/YOLOX) [![arXiv](https://img.shields.io/badge/arXiv-2107.08430-b31b1b.svg)](https://arxiv.org/abs/2107.08430)|
+| [YOLOR Object Detection](https://github.com/roboflow-ai/notebooks/blob/main/notebooks/train-yolor-object-detection-on-custom-data.ipynb) | [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/roboflow-ai/notebooks/blob/main/notebooks/train-yolor-object-detection-on-custom-data.ipynb) [![Kaggle](https://kaggle.com/static/images/open-in-kaggle.svg)](https://kaggle.com/kernels/welcome?src=https://github.com/roboflow-ai/notebooks/blob/main/notebooks/train-yolor-object-detection-on-custom-data.ipynb)  | [![Roboflow](https://raw.githubusercontent.com/roboflow-ai/notebooks/main/assets/badges/roboflow-blogpost.svg)](https://blog.roboflow.com/train-yolor-on-a-custom-dataset) [![YouTube](https://badges.aleen42.com/src/youtube.svg)](https://www.youtube.com/watch?v=sZ5DiXDOHEM) | [![GitHub](https://badges.aleen42.com/src/github.svg)](https://github.com/roboflow-ai/yolor) [![arXiv](https://img.shields.io/badge/arXiv-1506.0264-b31b1b.svg)](https://arxiv.org/abs/1506.0264)|
```

Comment:
Same here. https://arxiv.org/abs/1506.0264 and https://arxiv.org/abs/2107.0843 are wrong papers.